docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -110,6 +110,9 @@ is used.
   * - ``iceberg.max-partitions-per-writer``
     - Maximum number of partitions handled per writer.
     - 100
+  * - ``iceberg.orc.bloom-filters.enabled``
```

Comment:
per https://github.com/trinodb/trino/pull/11732/#discussion_r869112146, this will be `hive.orc ...`

Reply:
Based on this comment - https://github.com/trinodb/trino/pull/11732#discussion_r869980722

Reply:
Done.